master/version/version.go

```diff
@@ -8,3 +8,7 @@ const Unset = "unknown"
 // WARN: if you move it to a different package, you need to change the linked
 // path in the make file and CI.
 var Version = Unset
+
+// IsEE returns true if this is the enterprise codebase. It could parse the version, but the dev branch
+// of EE doesn't provide any differentiation from OSS.
+var IsEE = false
```

Comment:
I'm trying to think about where else we have this behavior and for rbac stuff we check the config of rbac enabled

the one other place I saw was in master info we return ```Branding: "determined"/"hpe"``` for oss/ee

so maybe that code could be updated to be identical using IsEE?

Reply:
I thought about that, but I think "IsEE" is saying something about which version of the codebase we are in, and "branding" is saying something about how the UI should be skinned, and I think we should keep those separate.